### PR TITLE
fix richeditor ul & ol padding on rtl

### PR DESCRIPTION
**Pull request by aymanalareqi**
this PR fix issue on the following image
rtl richeditor padding issue

### CODE REVIEW

File: packages/forms/resources/css/components/rich-editor.css [hunk 1]
```diff
@@ -22,3 +22,15 @@
 trix-toolbar .filament-forms-rich-editor-component-toolbar-button.trix-active {
     @apply bg-primary-600 text-white;
 }
+
+[dir='rtl'] .prose :where(ul):not(:where([class~='not-prose'] *)),
```
Comment (zepfietje) [inline packages/forms/resources/css/components/rich-editor.css:26]: @danharrin, this CSS isn't scoped to the rich-editor in any way... sure this won't cause issues in other places?

Comment (danharrin) [reply packages/forms/resources/css/components/rich-editor.css:26]: true

Comment (danharrin) [reply packages/forms/resources/css/components/rich-editor.css:26]: should be fixed

Comment (zepfietje) [reply packages/forms/resources/css/components/rich-editor.css:26]: @danharrin, could you revert this PR?
Then @aymanalareqi can attempt a new PR with a correct fix?

Comment (zepfietje) [reply packages/forms/resources/css/components/rich-editor.css:26]: Never mind, I didn't read your "should be fixed" as "has been fixed". 😄

**Pull request merged**